MAVProxy/modules/mavproxy_chat/chat_openai.py

```diff
@@ -99,6 +100,23 @@ def set_api_key(self, api_key_str):
         self.assistant = None
         self.assistant_thread = None
 
+    # cancel the active run
+    def cancel_run(self):
+        # check the active thread and run
+        if (self.assistant_thread and self.latest_run is not None):
+            if (self.latest_run.status != "completed" and "cancelled"):
```

Comment:
Oops, I think my comment was posted somewhere else.  In any case, I think the 2nd argument looks incorrect in this "if" statement.

Reply:
Sure! I got it.